addons/l10n_co_pos/views/views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="pos_config_view_form" model="ir.ui.view">
+        <field name="name">pos.config.form.view.inherit.l10n_co_pos</field>
+        <field name="model">pos.config</field>
+        <field name="inherit_id" ref="point_of_sale.pos_config_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='order_reference']" position="attributes">
```

Comment:
the field order_reference does not exist remove this xpath

Reply:
I've used ID 'order_reference' to remove group_no_one